torchgeo/datasets/dfc2022.py

```diff
@@ -0,0 +1,354 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""2022 IEEE GRSS Data Fusion Contest (DFC2022) dataset."""
+
+import glob
+import os
+from typing import Callable, Dict, List, Optional, Sequence
+
+import matplotlib.pyplot as plt
+import numpy as np
+import rasterio
+import torch
+from matplotlib import colors
+from rasterio.enums import Resampling
+from torch import Tensor
+
+from .geo import VisionDataset
+from .utils import check_integrity, extract_archive, percentile_normalization
+
+
+class DFC2022(VisionDataset):
+    """DFC2022 dataset.
+
+    The `DFC2022 <https://www.grss-ieee.org/community/technical-committees/2022-ieee-grss-data-fusion-contest/>`_
+    dataset is used as a benchmark dataset for the 2022 IEEE GRSS Data Fusion Contest
+    and extends the MiniFrance dataset for semi-supervised semantic segmentation.
+    The dataset consists of a train set containing labeled and unlabeled imagery and an
+    unlabeled validation set. The dataset can be downloaded from the
+    `IEEEDataPort DFC2022 website <https://ieee-dataport.org/competitions/data-fusion-contest-2022-dfc2022/>`_.
+
+    Dataset features:
+
+    * RGB aerial images at 0.5 m per pixel spatial resolution (~2,000x2,0000 px)
+    * digital elevation models (DEM) at 1 m per pixel spatial resolution (~1,000x1,0000 px)
+    * 16 land use/land cover pixelwise labels at 0.5 m per pixel spatial resolution (~2,000x2,0000 px)
+    * Images collected from the `IGN BD ORTHO database <https://geoservices.ign.fr/documentation/donnees/ortho/bdortho/>`_
+    * DEMs collected from the `IGN RGE ALTI database <https://geoservices.ign.fr/documentation/donnees/alti/rgealti/>`_
+    * Labels collected from the `UrbanAtlas 2012 database <https://land.copernicus.eu/local/urban-atlas/urban-atlas-2012/view/>`_
```

Comment:
A lot of these lines are way too long and should be wrapped onto a second line

Reply:
I've wrapped them but they are still too long